spec.yaml

```diff
@@ -84,29 +82,35 @@ workflow:
     id: sort_traj_speed
     task: sort_values
     partial:
-      column_name: "speed_bins"
-      ascending: True
+      column_name: speed_bins
+      ascending: false
```

Comment:
Confused why this is changed from `True` to `false` 🤔

Reply:
This experiment clarifies things ... it seems `dasel` roundtrips `True` -> `false`, but `true` is preserved on rt as `true`:

```console
➜  cat example.yaml
partial:
  column_name: whatever
  ascending: True
➜  dasel put -f example.yaml -t string -v whatever 'partial.column_name'
➜  cat example.yaml
partial:
  column_name: whatever
  ascending: false
➜  cat example2.yaml
partial:
  column_name: whatever
  ascending: true
➜ dasel put -f example2.yaml -t string -v whatever 'partial.column_name'
➜ cat example2.yaml
partial:
  column_name: whatever
  ascending: true

```

Reply:
**_Perhaps this is to be expected, according to Claude..._**


In standard YAML, `True` (with a capital T) is typically not considered a canonical boolean true value. While some YAML parsers might accept it due to Python-like syntax influence, it's not part of the official YAML specification.

The recommended boolean true values are:
- `true` (lowercase)
- `yes`
- `on`

If you're using a specific programming language's YAML parser (like PyYAML), it might accept `True`, but for maximum compatibility across different YAML parsers, it's best to stick to the lowercase `true`.

Here's a safe example:

```yaml
# Recommended way
is_active: true
is_enabled: yes
```

If you're unsure about a specific parser's behavior, it's always best to test or consult the documentation for that particular YAML implementation.